module/network/HTTPRequest.py

```diff
@@ -178,16 +192,16 @@ def setRequestContext(self, url, get, post, referer, cookies, multipart=False):
         if post:
             self.c.setopt(pycurl.POST, 1)
             if not multipart:
-                if type(post) == unicode:
-                    post = str(post) #unicode not allowed
-                elif type(post) == str:
+                #if type(post) == unicode:
+                #    post = str(post) #unicode not allowed
+                if type(post) == str:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)